TypeScript.py

```diff
@@ -439,6 +445,49 @@ def checkUpdateView(view):
         if cli.reloadRequired(view):
             reloadBuffer(view, clientInfo)
 
+# these three methods work with signature help
+def handle_signature_help(signature_help_response):
+    if signature_help_response.success:
```

Comment:
I feel like if you negate the conditions and bail out (i.e. `not signature_help_response.success`), you end up with something less nested and more readable